Add new example, meteor-site, to react-server examples

emecell opened the pull request:
This is from the talk given at Redfin on 11/9.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/redfin/react-server?pullRequest=750) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/redfin/react-server?pullRequest=750) before we can accept your contribution.<br/><hr/>**Robert Law** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

emecell commented on .eslintrc:
```diff
@@ -43,7 +43,8 @@
       2,
-      "tab"
+      "tab",
+      { SwitchCase: 1 }
```
I could drop this but otherwise it requires switch statements to be formatted like:
```
switch(a) {
case "blah":
default:
    return true;
}
```

doug-wade commented on packages/react-server-examples/meteor-site/package.json:
```diff
@@ -0,0 +1,29 @@
+    "superagent": "1.8.4"
+  },
+  "devDependencies": {
+    "babel-preset-react-server": "0.4.4"
```
You're missing `react-server-cli` as a dev dependency, so `npm start` horks if you don't have react-server-cli installed globally (which is nice if you have multiple react-server projects using different versions of the cli).

emecell commented:
@doug-wade I think you might have an older version of Chrome: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find#Browser_compatibility

I just switched it to use lodash's find instead though.

I updated the dependencies to include the `react-server-cli`.

doug-wade commented:
Got it up and running locally -- it's pretty slick.  Thanks @emecell
